frontend/public/components/machine-set.tsx

```diff
@@ -276,6 +317,19 @@ export const MachineSetList: React.SFC = (props) => {
   };
 
   const MachineSetTableRow: RowFunction<MachineSetKind> = ({ obj, index, key, style }) => {
+    const relatedMachines =
+      isMachineLoaded &&
+      machines.filter(
+        (machine) =>
+          machine.metadata.labels['machine.openshift.io/cluster-api-machineset'] ===
```

Comment:
I'd add optional chaining here just in case since the API doesn't require `labels` to be defined.

```suggestion
          machine.metadata.labels?.['machine.openshift.io/cluster-api-machineset'] ===
```